docs/bias_correction.rst

```diff
@@ -1,22 +1,84 @@
+
+`Callable workflow`_ : ``apply_annual_bmorph``, ``apply_interval_bmorph``
 
 Spatially Consistent Bias Correction: Univariate (SCBC_U)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Univariate Spatially Consistent Bias Correction (SCBC_U) aims to address IBC's inability to correct flows at non-gauge sites where reference timeseries do not exist. Spatial consistency is conserved by performing bias corrrections at every river segement, or `seg <data.rst/Common Naming Conventions>`_, and then rerouting the corrected flows through `mizuroute <https://mizuroute.readthedocs.io/en/latest/`_. Reference data for each seg that is not a gauge site is done by creating proxy reference data for each seg from upstream and downstream proxy gauge flows that can be combinded, or blended, together to create what the reference flow data for that seg should look like, as described in `Spatial Conistency: Reference Site Selection & CDF Blend Factor <data.rst/Spatial Consistency: Reference Site Selection & CDF Blend Factor>`_. 
+
+`Callable workflow`_ : ``apply_annual_blendmorph``, ``apply_interval_blendmorph``
 
 Spatially Consistent Bias Correction: Conditioned (SCBC_C)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conditioned Spatially Consistent Bias Correction (SCBC_C) combines the meterlogic conditioning elements of `IBC_C <Independent Bias Correction: Conditioned (IBC_C)>`_ with the spatial consistency of `SCBC_U <Spatially Consistent Bias Correction: Univariate (SCBC_U)>`_. This implementation of SCBC factors in meterologic variables given into the formulation of refernce flows for each seg to be corrected to. Defined by the hydrologic response units, or <hru's <data.rst/Common Naming Conventions>`_, they impact, meterologic data is mappable to each seg within the watershed topology. In `IBC_C <Independent Bias Correction: Conditioned (IBC_C)>`_, only the data mapped to gauge sites would be used in bias correction, whereas SCBC_C can utilize meterologic data across the watershed as it incoporates all defined segs. 
+
+`Callable workflow`_ : ``apply_annual_blendmorph``, ``apply_interval_blendmorph``
+
+.. _`Callable workflow`: https://bmorph.readthedocs.io/en/develop/api.html#module-bmorph.core.workflows
+
+bmorph Parameters
+-----------------
+
+Conditioning: edcdfm vs mdcdedcdfm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+EDCDFm
+""""""
+
+Before describing how conditioning is integrated into ``bmorph`` bias correction, we need to discuss the standard method. 
+``bmorph`` implements part of PresRat bias correction from Pierce et al. (2015), which is an extension of Equidistant quantile matching (EDCDFm) technique of Li et al. (2010). ``bmorph`` uses the amended EDCDFm to compute multiplicative changes in the quantiles of a Cumulative Distribution Fuction (CDF). Here, only flow values are used to construct the CDFs. 
+
+.. image:: Figures/Quantile_Map_Diagram.png
```

Comment:
This is actually not a diagram of EDCDFm, but just quantile mapping. The Pierce paper has a diagram of how EDCDFm works if we need to include one.

Reply:
I don't think I ever referred to the figure as being Quantile Mapping, but I see where that is misleading. I think it'd be helpful to include a diagram, so I'll grab that from the Pierce paper and site it.